style.css

```diff
@@ -0,0 +1,646 @@
+.nav-buttons-2 a {
+	margin-left: 20px;
+	color: #fbfcfc;
+	font-size: 20px;
+}
+
+.name a:link { 
+	text-decoration: none;
+}
+
+.name a:visited { 
+	text-decoration: none; 
+}
+
+.name a:hover { 
+	text-decoration: none;
+	color: #1d7e2b;
+}
+
+.name a:active { 
+	text-decoration: none; 
+}
+
+a:link { 
+	text-decoration: none;
+}
+
+a:visited { 
+	text-decoration: none; 
+}
+
+a:hover { 
+	text-decoration: none;
+	color: #1d7e2b;
+}
+
+a:active { 
+	text-decoration: none; 
+}
+
+.intro {
+	background-image: url("images/raven2.jpeg");
+	background-position: center center;
+	background-repeat: no-repeat;
+	background-attachment: fixed;
+	background-size: cover;
+	background-color: #d0d1d3;
+	width: 100%;
+	height: 700px;
+	display: flex;
+	justify-content: flex-end;
+	align-items: center;
+	padding-right: 20%;
+	padding-bottom: 200px;
+	padding-top: 360px;
+	font-family: 'Ibarra Real Nova', serif;
+}
+	
+.blurb h1 {
+	font-size: 50px;
+	color: #050608;;
+}
+
+.blurb h1 :hover {
+	color: red;
+}
+
+
+.blurb h3 {
+	font-size: 30px;
+	color: #050608;
+}
+
+/* ABOUT SECTION*/
+
+.about {
+	background-image: url("images/about-photo.jpg");
+	background-position: center center;
+	background-repeat: no-repeat;
+	background-attachment: fixed;
+	background-size: cover;
+	background-color: #d0d1d3;
+}
+
+.central-container {
+	height: 100px;
+}
+
+section .title-border {
+	display: flex;
+	justify-content: center;	
+	background-color: #191c1d;
+	width: 100%;
+}
+
+section div h2 {
+	background-color: #191c1d;
+	color: #f3f4f6;
+	padding: 20px;
+
+}
+
+.about-container {
+	display: flex;
+	justify-content: center;
+	padding-top: 50px;
+	padding-bottom: 50px;
+}
+
+.text {
+	width: 650px;
+	height: 540px;
+	font-size: 1.5rem;
+	border: solid #191c1d 10px;
+	padding: 35px;
+	background-color:rgba(204, 206, 211, 0.7);
+	font-family:  'Ibarra Real Nova', serif;
+	text-align: center;
+	border: solid #191c1d 5px;
+	border-right: 0.5px solid transparent;
+}
+
+.josh-photo {
+	width: 650px;
+	height: 540px;
+	background-color:rgba(204, 206, 211, 0.7);
+	padding: 60px;
+	border: solid #191c1d 5px;
+	border-left: solid transparent 0.5px;
+}
+
+
+/* PORTFOLIO SECTION*/
+
+
+.portfolio {
+	background-image: url("images/about-photo.jpg");
+	background-position: center center;
+	background-repeat: no-repeat;
+	background-attachment: fixed;
+	background-size: cover;
+	background-color: #d0d1d3;
+}
+
+.portfolio-container {
+	display: flex;
+	justify-content: center;
+	padding-top: 50px;
+	padding-bottom: 50px;
+}
+
+.project-1 {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	margin-top: 100px;
+	border: solid #191c1d 5px;
+	width: 1300px;
+	background-color: rgba(204, 206, 211, 0.7);
+	padding-right: 30px;
+}
+
+.project-photo {
+	width: 380px;
+	height: auto;
+	opacity: 0.6;
+	padding-left: 20px;
+}
+
+.project-photo:hover {
+	opacity: 1;
+}
+
+.project-description {
+	display: flex;
+	flex-direction: column;
+	width: 25%;
+}
+
+.project-2 {
```

Comment:
this project code is repeated here and in media queries - just assign the styles to a single class and give it to all of them. If you need a specific one to do something different then create an additional new class for that one to override the styles